Sources/core/objects/OnMapBase.h

```diff
@@ -147,6 +147,7 @@ class IOnMapBase: public IDraw
     virtual void UpdatePassable() {}
     virtual IdPtr<ITurf> GetTurf();
     virtual void SetTurf(IdPtr<ITurf> turf);
+    IdPtr<CubeTile> GetRoot();
```

Comment:
It should not be `CubeTile`, but `IOnMapBase`. `GetRoot` not always will return `CubeTile`.